java-parser/src/test/java/org/archcnl/javaparser/visitors/ThrowStatementVisitorTest.java

```diff
@@ -52,6 +55,17 @@ public void testSimpleNames() throws FileIsNotAJavaClassException, FileNotFoundE
         assertTrue(exceptions.contains("IllegalStateException"));
     }
 
+    @Test
+    public void testTryCatchThrow() throws FileIsNotAJavaClassException, FileNotFoundException {
+        final CompilationUnit unit = CompilationUnitFactory.getFromPath(pathToTryCatchThrowClass);
+
+        unit.accept(visitor, null);
+
+        final List<Type> exceptions = visitor.getThrownExceptionType();
+
+        assertEquals(0, exceptions.size());
```

Comment:
Maybe add a comment explaining why 0 is expected

Reply:
I've added a description to the test method. You might want to take a look, if the description is usable and understandable.